pallet/src/weights.rs

```diff
@@ -38,52 +38,52 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
-	/// Storage: `MoveModule::ChoreOnIdleStorage` (r:1 w:1)
-	/// Proof: `MoveModule::ChoreOnIdleStorage` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// Storage: `MoveModule::VMStorage` (r:3 w:2)
+	/// Storage: `MoveModule::VMStorage` (r:2 w:1)
 	/// Proof: `MoveModule::VMStorage` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// The range of component `n` is `[0, 7]`.
+	/// The range of component `n` is `[0, 9]`.
 	fn execute(n: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `8700`
-		//  Estimated: `14577 + n * (812 ±0)`
-		// Minimum execution time: 36_000_000 picoseconds.
-		Weight::from_parts(552_887_566, 0)
-			.saturating_add(Weight::from_parts(0, 14577))
-			.saturating_add(T::DbWeight::get().reads(6))
+		//  Measured:  `9476`
+		//  Estimated: `17211 + n * (1491 ±26)`
+		// Minimum execution time: 33_000_000 picoseconds.
+		Weight::from_parts(33_000_000, 0)
+			.saturating_add(Weight::from_parts(0, 17211))
+			// Standard Error: 289_551_375
+			.saturating_add(Weight::from_parts(5_224_318_063, 0).saturating_mul(n.into()))
+			.saturating_add(T::DbWeight::get().reads(5))
 			.saturating_add(T::DbWeight::get().writes(3))
-			.saturating_add(Weight::from_parts(0, 812).saturating_mul(n.into()))
+			.saturating_add(Weight::from_parts(0, 1491).saturating_mul(n.into()))
 	}
 	/// Storage: `MoveModule::VMStorage` (r:2 w:1)
 	/// Proof: `MoveModule::VMStorage` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// The range of component `n` is `[0, 3]`.
 	fn publish_module(n: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `111 + n * (3506 ±0)`
-		//  Estimated: `2180 + n * (4531 ±116)`
-		// Minimum execution time: 36_000_000 picoseconds.
-		Weight::from_parts(25_415_305, 0)
-			.saturating_add(Weight::from_parts(0, 2180))
-			// Standard Error: 1_274_995
-			.saturating_add(Weight::from_parts(132_265_386, 0).saturating_mul(n.into()))
+		//  Measured:  `111 + n * (3842 ±0)`
+		//  Estimated: `2081 + n * (4852 ±124)`
+		// Minimum execution time: 38_000_000 picoseconds.
+		Weight::from_parts(27_213_142, 0)
+			.saturating_add(Weight::from_parts(0, 2081))
+			// Standard Error: 1_232_515
+			.saturating_add(Weight::from_parts(127_581_233, 0).saturating_mul(n.into()))
 			.saturating_add(T::DbWeight::get().reads(1))
 			.saturating_add(T::DbWeight::get().writes(1))
-			.saturating_add(Weight::from_parts(0, 4531).saturating_mul(n.into()))
+			.saturating_add(Weight::from_parts(0, 4852).saturating_mul(n.into()))
 	}
 	/// Storage: `MoveModule::VMStorage` (r:1 w:1)
 	/// Proof: `MoveModule::VMStorage` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	fn publish_module_bundle() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `111`
 		//  Estimated: `3576`
-		// Minimum execution time: 161_000_000 picoseconds.
-		Weight::from_parts(168_000_000, 0)
+		// Minimum execution time: 158_000_000 picoseconds.
```

Comment:
We can discuss this one on Slack

Reply:
Implemented a parameter-based benchmarking for this extrinsic 👍